docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -0,0 +1,259 @@
+    of the variation in the model results comes from the model.
+
+Classification
+--------------
+In general, the common classification metrics are based on the concept of comparing the predicted label with the ground
+truth label and counting the matches and mismatches between them.
+
+This is usually done with the assistance of a confusion matrix. If you are not familiar with the concept of confusion
+matrix, check out this `link <https://en.wikipedia.org/wiki/Confusion_matrix>`__ .
+
+.. image:: /_static/images/general/truth_table.png
+   :alt: Truth Table
+   :align: center
+
+Truth table by Sol Yarkoni
+
+*   `Accuracy <https://developers.google.com/machine-learning/crash-course/classification/accuracy>`__ - how many
+    samples were classified correctly out of the total number of samples. Though very intuitive, it can often be
+    misleading, especially for imbalanced data.
+*   `Precision <https://en.wikipedia.org/wiki/Precision_and_recall>`__ - how many samples were classified correctly as
+    positive out of the total number of samples classified as positive. Can be also viewed as the fraction of relevant
+    samples out of the samples spotted by the model.
+*   `Recall <https://en.wikipedia.org/wiki/Precision_and_recall>`__ /
+    `Sensitivity <https://en.wikipedia.org/wiki/Sensitivity_and_specificity>`__ / TPR - how many samples were classified
+    correctly as positive out of the total number of positive samples. Can be also viewed as the fraction of the
+    relevant samples that were spotted by the model.
+
+
+.. image:: /_static/images/general/PrecisionRecall_wikipedia.svg.png
+   :alt: Precision and recall
+   :align: center
+
+Precision and Recall by Walber - Own work, CC BY-SA 4.0,
+`source <https://commons.wikimedia.org/w/index.php?curid=36926283>`__
+
+
+*   `Specificity <https://en.wikipedia.org/wiki/Sensitivity_and_specificity>`__/ TNR - how many samples were classified
+    correctly as negative out of the total number of negative samples. Complementary to sensitivity.
+
+.. image:: /_static/images/general/Sensitivity_and_specificity_wikipedia.svg.png
+   :alt: Specificity and sensitivity
+   :align: center
+
+Specificity and Sensitivity by FeanDoe - Modified version from Walber&#039;s Precision and Recall
+`source <https://commons.wikimedia.org/w/index.php?curid=94134880>`__
+
+*   `F-1 <https://en.wikipedia.org/wiki/F-score>`__ - combines the precision and recall into one metric by taking their
+    harmonic mean. More robust to class imbalance than accuracy.
+*   `AUC <https://developers.google.com/machine-learning/crash-course/classification/roc-and-auc>`__  - The area under
+    the curve of the ROC graph. For the previous classification metrics, a threshold on the model output was chosen
+    above which the sample is classified as positive and below as negative. The AUC takes into account all possible
+    thresholds.
+
+Object Detection
```

Comment:
Would explain how FP, TP, FN are calculated for object detection (what JKL did)
Also add them as metric options

Reply:
@Nadav-Barak explained in the link, WDYM as metric options?

Reply:
IMO I would explain them specifically here in a different paragraph as they can be regarded as metric on their on